Description: Link with -lflint-arb instead of -larb
 The name of the arb library is libflint-arb in Debian.
Author: Tobias Hansen <thansen@debian.org>
Forwarded: not-needed


--- a/sage/src/module_list.py
+++ b/sage/src/module_list.py
@@ -80,7 +80,7 @@
     "ec", "ecm",
 ] + aliases["LINBOX_LIBRARIES"] + aliases["FFLASFFPACK_LIBRARIES"] + aliases["GSL_LIBRARIES"] + [
     "pari", "flint", "ratpoints", "ecl", "glpk", "ppl",
-    "arb", "mpfi", "mpfr", "mpc", "gmp", "gmpxx",
+    "flint-arb", "mpfi", "mpfr", "mpc", "gmp", "gmpxx",
     "brial",
     "brial_groebner",
     "m4rie",
@@ -772,7 +772,7 @@
 
     Extension("sage.matrix.matrix_complex_ball_dense",
               ["sage/matrix/matrix_complex_ball_dense.pyx"],
-              libraries=['arb', 'mpfi', 'mpfr'],
+              libraries=['flint-arb', 'mpfi', 'mpfr'],
               language = "c++"),
 
     Extension('sage.matrix.matrix_complex_double_dense',
--- a/sage/src/sage/libs/arb/acb.pxd
+++ b/sage/src/sage/libs/arb/acb.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from sage.libs.arb.types cimport *
 from sage.libs.flint.types cimport fmpz_t, fmpq_t
--- a/sage/src/sage/libs/arb/acb_hypgeom.pxd
+++ b/sage/src/sage/libs/arb/acb_hypgeom.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from sage.libs.arb.types cimport *
 
--- a/sage/src/sage/libs/arb/acb_modular.pxd
+++ b/sage/src/sage/libs/arb/acb_modular.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from sage.libs.arb.types cimport *
 from sage.libs.flint.types cimport fmpz_poly_t
--- a/sage/src/sage/libs/arb/arb.pxd
+++ b/sage/src/sage/libs/arb/arb.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from sage.libs.arb.types cimport *
 from sage.libs.flint.types cimport fmpz_t, fmpq_t
--- a/sage/src/sage/libs/arb/arf.pxd
+++ b/sage/src/sage/libs/arb/arf.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from sage.libs.arb.types cimport *
 from sage.libs.gmp.types cimport mpz_t
--- a/sage/src/sage/libs/arb/bernoulli.pxd
+++ b/sage/src/sage/libs/arb/bernoulli.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from ..flint.types cimport fmpq_t, ulong
 
--- a/sage/src/sage/libs/arb/mag.pxd
+++ b/sage/src/sage/libs/arb/mag.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from sage.libs.arb.types cimport *
 from sage.libs.flint.types cimport fmpz_t, fmpq_t
--- a/sage/src/sage/libs/arb/acb_poly.pxd
+++ b/sage/src/sage/libs/arb/acb_poly.pxd
@@ -1,4 +1,4 @@
-# distutils: libraries = gmp flint arb
+# distutils: libraries = gmp flint flint-arb
 
 from sage.libs.arb.types cimport *
 from sage.libs.flint.types cimport fmpz_poly_t, fmpq_poly_t
